commit 20f3a2abc884be5432f9dc4445d0a4833998b679 (refs/changes/30/3330/1)
Author: Kody A Kantor <kody.kantor@gmail.com>
Date:   2018-02-08T21:14:01+00:00 (1 year, 8 months ago)
    
    MANTA-3575 increase config-agent poll interval to 120 seconds
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/deps/manta-scripts b/deps/manta-scripts
index 253ae31..dc593cd 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit 253ae31df06a8b5a3d520e3874d0eb060dda4fa3
+Subproject commit dc593cd90475e391c6fce874130f5d522dc0cc54
